wandb/internal/sender.py

```diff
@@ -523,7 +523,7 @@ def send_artifact(self, data):
         saver.save(
             type=artifact.type,
             name=artifact.name,
-            metadata=json.loads(artifact.metadata),
+            metadata=json.loads(artifact.metadata or "{}"),
```

Comment:
Thanks for fixing!

Reply:
Actually, can we make this: `if artifact.metadata then json.loads(artifact.metadata) else None`? Not a huge deal, but we'll end up setting an empty metadata in all the artifacts instead of just leaving them `NULL`.